core/scripts/local/AdvanceEMP.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * This script advances time in the EMP forward by some specified amount of seconds (or one hour and one second by default).
+ *
+ * `truffle exec ./scripts/local/AdvanceEMPTime --time 10000`
+ * Advances time by 10 seconds
+ */
+const { toBN } = web3.utils;
+const argv = require("minimist")(process.argv.slice(), { string: ["time"] });
+
+// Deployed contract ABI's and addresses we need to fetch.
+const Governor = artifacts.require("Governor");
+const Registry = artifacts.require("Registry");
+const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
+
+// Contracts we need to interact with.
+let registry;
+let emp;
+
+/** ***************************************************
+ * Main Script
+ /*****************************************************/
+const advanceTime = async callback => {
+  try {
+    registry = await Registry.deployed();
+    const contractAddresses = await registry.getAllRegisteredContracts();
+
+    // Query all registered EMP's.
+    for (const address of contractAddresses) {
+      // The governor is always registered as a contract, but it isn't an ExpiringMultiParty.
+      if (address !== Governor.address) {
+        emp = await ExpiringMultiParty.at(address);
+
+        // Advance time
+        let leapForward = argv.time ? argv.time : 3601;
+        let currentTime = await emp.getCurrentTime();
+        let newTime = toBN(currentTime.toString()).add(toBN(leapForward.toString()));
+        await emp.setCurrentTime(newTime.toString());
```

Comment:
here you are only advancing time within the emp. this _should_ be fine but if in testing a vote needs to happen on price (dispute, settlement, emergencyshutdown) then the DVM time will be disjoint from the emp time. Quick, dirty fix is to set the DVM time here as well. Longer term solution is outlined in #1047

Reply:
I've been using the mock oracle, but I suppose I can grab the relevant oracle and set its time forward too.

Reply:
cool. it should just advance all time consistently I think